src/overlays/actors/ovl_Bg_Bdan_Switch/z_bg_bdan_switch.c

```diff
@@ -59,17 +59,31 @@ extern CollisionHeader D_06005CF8;
 extern Gfx D_060061A0[];
 extern Gfx D_06005A20[];
 
-static ColliderJntSphItemInit sJntSphItemsInit[] = {
+static ColliderJntSphElementInit sJntSphElementsInit[] = {
```

Comment:
While on the topic of collider inits, I want to suggest that we make defines for the elements count of each collider. It would be in the header of the actor so it can be used in the struct for the size of the elements array, as well as the size of the init array in the C file. This would ensure they are always the same size, mostly for modding purposes.

This also doesn't have to happen in this set of changes tho, just bringing it up so we can start thinking about it.